scripts/utility.js

```diff
@@ -0,0 +1,7 @@
+export function getElement(dataJS, target = document) {
+  return target.querySelector(`[data-js=${dataJS}]`)
```

Comment:
I see, you already adapted the information for the pages we read this weekend. Nice!